src/PhpSpec/Console/IO.php

```diff
@@ -342,4 +342,25 @@ public function getBlockWidth()
         }
         return $width;
     }
+
+    /**
+     * @param string $message
+     * @param int|null $indent
+     */
+    public function writeError($message, $indent = null)
+    {
+        $message = wordwrap($message, $this->getBlockWidth() - 4, "\n", true);
```

Comment:
if we want to keep it more generic, it should be `$indent * 2` tbh, then the error message is displayed with the same padding on both sides